prov/rxm/src/rxm_conn.c

```diff
@@ -1430,6 +1450,11 @@ rxm_conn_connect(struct util_ep *util_ep, struct rxm_cmap_handle *handle,
 		},
 	};
 
+	assert(sizeof(uint32_t) == sizeof(cm_data.connect.eager_size));
+	assert(sizeof(uint32_t) == sizeof(cm_data.connect.rx_size));
+	assert(rxm_ep->rxm_info->tx_attr->inject_size <= (uint32_t)-1);
+	assert(rxm_ep->msg_info->rx_attr->size <= (uint32_t)-1);
```

Comment:
I think you can use UINT32_MAX

Reply:
I can do this in a follow-up so that we don't have to re-run the CI again.